WalletWasabi.Gui/Converters/MaxClearBoolStringConverter.cs

```diff
@@ -10,19 +10,14 @@ public class MaxClearBoolStringConverter : IValueConverter
 	{
 		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
 		{
-			if (value is bool boolean)
+			if (value is bool isMax)
 			{
-				if (boolean)
-				{
-					return "Clear";
-				}
-				else
-				{
-					return "Max";
-				}
+				return isMax ? "Clear" : "Max";
```

Comment:
This is just a comment: we have this converters from `bool` to `Clear/Max`, `On/Off` and probably other. They are all the same, I will change this.